src/components/UserInfo.js

```diff
@@ -1,7 +1,14 @@
 export default class UserInfo {
-  constructor(userNameSelector, userDescriptionSelector) {
+  constructor(
+    userNameSelector,
+    userDescriptionSelector,
+    profilePictureSelector
+  ) {
     this._userNameEl = document.querySelector(userNameSelector);
     this._userJobEl = document.querySelector(userDescriptionSelector);
+    this._profilePictureSelector = document.querySelector(
```

Comment:
`_profilePictureSelector`

Word `selector` means a **string** that is used for searching elements in the DOM